### DISCUSSION TITLE
new hack

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#962] [(C) Suggestions]
i think it would be cool if you could morph in to titans "client side only" of course allso i think you should add an set damedge and a set health hack please and thank u

**Comment by hostedposted** [comment]
Impossible. I tried and my icon disappeared.

**Reply by ghost** [reply]
oh

**Reply by ghost** [reply]
what about aincient can i morph into thtat?
